cgi/import_products_categories_from_public_database.pl

```diff
@@ -55,23 +55,15 @@
 
 my $title = lang("import_products_categories_from_public_database");
 my $html = '';
+my $js = '';
```

Comment:
I don't think the $js variable is used?

Reply:
Oh yeah sorry, added it for the next process file. Removed now.